common/views/components/PrototypeSearchForm/PrototypeSearchForm.tsx

```diff
@@ -99,6 +102,16 @@ const PrototypeSearchForm: FunctionComponent<Props> = ({
       );
   }
 
+  useEffect(() => {
```

Comment:
This makes me a bit uneasy, do we know why rendering of createPortal doesn't trigger

Reply:
Mainly cause the props don't re-trigger the createPortal to replicate the nodes to the required place. It should trigger this without it but doesn't so im forcing it to trigger it. I have made sure it only gets trigger once also in the useEffect.. 

without the adhoc code, If i type something in input search fields it trigger the createPortal mainly cause it knows to rerender.

Reply:
as discussed, add additional comment on the reason for the solution